src/Snippet/CurrentDatabase.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\ClickHouseClient\Snippet;
```

Comment:
imo by bylo lepší mít jednu třídu kde tyhle snippety budou sdružený ... klidně je měj definovaný separe v souborech, ale pak mít jedno místo odkud se to dá volat ... když ti IDE napovídá tak snadno najdeš co hledáš, ale hledat to v classách je hroznej bordel

Reply:
je to documented + v jedny slozce, tak nevidim issue. navic vetsi granularita a SRP compliant

Reply:
Ani jedno ale neřeší problém s používáním ... chci reálně udělat něco ve stylu `$this->chClient->[ctrl+mezerník]`, abych viděl možnosti a z nich si vybrat ... tohle je sice granulární a srp compliant, ale totálně na hovno pro praktický použití. Proto říkám, klidně nechat jak je, ale přidat ještě classu co to sjednotí ... aby fungovalo tohle běžný použití

Reply:
no ale ja nechci cunit jenom proto, ze jsou nektery lidi liny.

je tu dokumentace, pokud chces prakticky pouziti. jeste v ni bude muset bejt napsany presne co to dela. urcite nechci lidi nechat poustet random query, co si nasli v autocomplete. aspon si nevydropej omylem db, jako se to stalo nam :D

Reply:
Sorry ale tohle je fakt shit no, bude se to používat na hovno. Nikdo nechce jít studovat dlouhou dokumentaci aby našel to co ostatní libs umí hned "z kódu"

Reply:
ostatni libs maj spatnej design a ja ho urcite nechci prejimat. proto jsem napsal tuhle.  
i kdyz by SRP neexistovalo, tak vytvorit jednu obrovskou snippet abomination je no no

documentation je uplne ok, staci aby [mela osnovu](https://github.com/simPod/grafana-json-datasource) a nemusis ji cist celou, jen relevantni cast

Reply:
nemyslím si že je to špatnej design ... ale whatever, používat to naštěstí nebudu a kdyžtak se tě budu ptát :D

Reply:
ja si od zacatku trochu myslim, ze se to ve vysledku stejne vyhodi po zkusenostech z puvodni knihovny. ale rikal jsem si, ze to zkusim